readme.md

```diff
@@ -218,6 +218,30 @@ queue.add(() => Promise.resolve());
 queue.add(() => Promise.resolve());
 queue.add(() => delay(500));
 ```
+#### idle
+
+Emitted when the queue becomes empty, and all promises have completed; `queue.size === 0 && queue.pending === 0`.
```

Comment:
```suggestion
Emitted every time the queue becomes empty, and all promises have completed; `queue.size === 0 && queue.pending === 0`.
```

To make it clear that it can be emitted multiple times.

I would also mention the difference with `.onIdle()`.